aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -456,9 +457,33 @@ module AptosFramework::Stake {
             };
             i = i + 1;
         };
+
+        sort_validators(&mut active_validators);
         validator_set.active_validators = active_validators;
     }
 
+    fun sort_validators(validators: &mut vector<ValidatorInfo>) {
```

Comment:
I thought you wanted to do this in a native function? This is fine too as I'm not sure if there's a big performance gain from that.